dask/array/core.py

```diff
@@ -3458,7 +3458,7 @@ def from_array(
 
     >>> import numpy as np
     >>> import dask.array as da
-    >>> x = np.random.random((100, 6))
+    >>> x = np.random.default_rng().random((100, 6))
```

Comment:
```suggestion
    >>> rng = np.random.default_rng()
    >>> x = rng.random((100, 6))
```
Same as my other non-blocking suggestion.